### PR TITLE
Update review summary header

### PR DESCRIPTION
**Pull request by nop33**
### Hiding the header information
image

#### Example 1
image
#### Example 2
image

### CODE REVIEW

File: indico/modules/events/abstracts/templates/abstract/judge.html [hunk 1]
```diff
@@ -19,11 +18,20 @@
                             'conflicting': conflicting_msg,
                             'negative': _("No reviewer proposed to accept this abstract"),
                             'mixed': _("Not all reviewers proposed to accept this abstract")} %}
+    {% set mapping_icon = {'positive': 'icon-checkmark',
+                           'conflicting': 'icon-checkmark',
+                           'negative': 'icon-cross',
+                           'mixed': 'icon-tilde',
+                           'in_progress': 'icon-circle-small',
+                           'not_started': 'icon-circle-small'} %}
```
Comment (nop33) [inline indico/modules/events/abstracts/templates/abstract/judge.html]: We have many mapping sets in this file (open the file to see the whole content). Any ideas as to how to simplify it?

Comment (OmeGak) [inline indico/modules/events/abstracts/templates/abstract/judge.html]: Not sure if any simplification won't end up just moving the complexity somewhere else. I don't think it's so ugly right now.

File: indico/modules/events/abstracts/views.py [hunk 2]
```diff
@@ -108,12 +108,15 @@ def render_abstract_page(abstract, view_class=None, management=False):
         review_form = build_review_form(abstract, reviewed_for_tracks[0])
     judgment_form = AbstractJudgmentForm(abstract=abstract, formdata=None)
     review_track_list_form = AbstractReviewedForTracksForm(event=abstract.event_new, obj=abstract, formdata=None)
+    convener_tracks = set(get_convener_tracks(abstract.event_new, session.user))
```
Comment (OmeGak) [inline indico/modules/events/abstracts/views.py]: We use `get_convener_tracks` here only and we don't really care about the order, so maybe it's worth returning a set directly.

File: indico/modules/events/abstracts/views.py [hunk 3]
```diff
@@ -108,12 +108,15 @@ def render_abstract_page(abstract, view_class=None, management=False):
...
+    visible_tracks = set(t for t in abstract.reviewed_for_tracks
```
Comment (OmeGak) [inline indico/modules/events/abstracts/views.py]:
```python
if abstract.can_judge(session.user, check_state=True):
    visible_tracks = abstract.reviewed_for_tracks
else:
    ...
```

Comment (OmeGak) [inline indico/modules/events/abstracts/views.py]: > Discussed IRL

**Pull request merged**